tests/base/mpi_noncontiguous_vector_02.mpirun=8.output

```diff
@@ -0,0 +1,111 @@
+DEAL:3::
+DEAL:3::
+DEAL:3::131 130 129 128 127 126 125 124 123 122 121 120 119 118 117 116 115 114 113 112 111 110 109 108 107 106 105 104 103 102 101 100 99 98 97 96 
+DEAL:3::
+DEAL:3::
+DEAL:3::140 128 116 104 92 80 68 56 44 32 20 8 
+DEAL:3::
+
+
+DEAL:4::72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 
+DEAL:4::96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 
+DEAL:4::
+DEAL:4::72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 
+DEAL:4::8 20 32 44 56 68 80 92 104 116 128 140 
+DEAL:4::
+DEAL:4::72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 
+DEAL:4::119 118 117 116 115 114 113 112 111 110 109 108 107 106 105 104 103 102 101 100 99 98 97 96 95 94 93 92 91 90 89 88 87 86 85 84 83 82 81 80 79 78 77 76 75 74 73 72 
+DEAL:4::
+DEAL:4::72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 
+DEAL:4::139 138 127 126 115 114 103 102 91 90 79 78 67 66 55 54 43 42 31 30 19 18 7 6 
+DEAL:4::
+
+
+DEAL:5::
+DEAL:5::108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 
+DEAL:5::
+DEAL:5::
+DEAL:5::9 21 33 45 57 69 81 93 105 117 129 141 
+DEAL:5::
+DEAL:5::
+DEAL:5::95 94 93 92 91 90 89 88 87 86 85 84 83 82 81 80 79 78 77 76 75 74 73 72 71 70 69 68 67 66 65 64 63 62 61 60 59 58 57 56 55 54 53 52 51 50 49 48 
+DEAL:5::
+DEAL:5::
+DEAL:5::137 136 125 124 113 112 101 100 89 88 77 76 65 64 53 52 41 40 29 28 17 16 5 4 
+DEAL:5::
+
+
+DEAL:6::90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 
+DEAL:6::120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 
+DEAL:6::
+DEAL:6::90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 
+DEAL:6::10 22 34 46 58 70 82 94 106 118 130 142 
+DEAL:6::
+DEAL:6::90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 
+DEAL:6::63 62 61 60 59 58 57 56 55 54 53 52 51 50 49 48 47 46 45 44 43 42 41 40 39 38 37 36 35 34 33 32 31 30 29 28 27 26 25 24 
+DEAL:6::
+DEAL:6::90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 
+DEAL:6::135 134 123 122 111 110 99 98 87 86 75 74 63 62 51 50 39 38 27 26 15 14 3 2 
+DEAL:6::
+
+
+DEAL:7::
+DEAL:7::132 133 134 135 136 137 138 139 140 141 142 143 
+DEAL:7::
+DEAL:7::
+DEAL:7::11 23 35 47 59 71 83 95 107 119 131 143 
+DEAL:7::
+DEAL:7::
```

Comment:
Is there additional value in recording the reference values for more than say 1 and 4 ranks? I mean in the sense of a probability that we could have those cases passing but the new infrastructure being broken. I'm asking because we tend to be conservative with tests using more than a handful of MPI ranks.

Reply:
I have reduced the number of processes to 5. This is a configuration that has an empty index set.